plugins/datadog-standalone/datadog-standalone.go

```diff
@@ -88,7 +88,8 @@ func datadogConnector(metrics []producers.MetricsMessage, c *cli.Context) error
 
 	result, err := postMetricsToDatadog(datadogURL, metrics)
 	if err != nil {
-		return err
+		log.Errorf("Unexpected error while processing DataDog response: %v", err)
+		return nil
```

Comment:
+1 to returning nil here.  It's a little unfortunate that if the connector func returns a non-nil err it causes the process to exit.  A future pull request should probably address that because it's janky to just return nil here if an error happens.